.golangci.yml

```diff
@@ -1,27 +1,59 @@
 # Copyright KubeArchive Authors
 # SPDX-License-Identifier: Apache-2.0
 ---
+version: '2'
 run:
-  timeout: 10m
   build-tags:
     - integration
 linters:
   enable:
     - bodyclose
     - containedctx
     - fatcontext
-    - goimports
```

Comment:
Just to make sure I get this, `goimports` is still in place but the new version required it as part of the `formatters`, isn't it?

Reply:
Yes, formatters are run in linting mode when doing `golangci-lint run`.  (from https://ldez.github.io/blog/2025/03/23/golangci-lint-v2/)